packages/tracing/src/browser/metrics.ts

```diff
@@ -134,6 +134,43 @@ export class MetricsInstrumentation {
 
     // Measurements are only available for pageload transactions
     if (transaction.op === 'pageload') {
+      // normalize applicable web vital values to be relative to transaction.startTimestamp
+
+      const timeOrigin = msToSec(performance.timeOrigin);
+
+      ['fcp', 'fp', 'lcp', 'ttfb'].forEach(name => {
+        if (!this._measurements[name] || timeOrigin >= transaction.startTimestamp) {
+          return;
+        }
+
+        // The web vitals, fcp, fp, lcp, and ttfb, all measure relative to timeOrigin.
+        // Unfortunately, timeOrigin is not captured within the transaction span data, so these web vitals will need
+        // to be adjusted to be relative to transaction.startTimestamp.
+
+        const oldValue = this._measurements[name].value;
+        const measurementTimestamp = timeOrigin + msToSec(oldValue);
+        // normalizedValue should be in milliseconds
+        const normalizedValue = (measurementTimestamp - transaction.startTimestamp) * 1000;
+
+        const delta = normalizedValue - oldValue;
+        logger.log(
+          `[Measurements] Normalized ${name} from ${this._measurements[name].value} to ${normalizedValue} (${delta})`,
+        );
+
+        this._measurements[name] = { value: normalizedValue };
+      });
+
+      if (this._measurements['mark.fid'] && this._measurements['fid']) {
+        // create span for FID
+
+        _startChild(transaction, {
+          description: 'first input delay',
+          endTimestamp: this._measurements['mark.fid'].value + msToSec(this._measurements['fid'].value),
+          op: 'web vitals',
```

Comment:
Should this be 'vitals' to match the more generic name we'll likely use in the future?

Reply:
Right now we've not committed to using that term as a generic umbrella.

We've only used "web vitals" thus far.

Reply:
Is the fact that the op is currently two words going to mean that when searching for it, you have to put it in quotes? (I would guess that the answer is yes.) If so, can we make this one word (even if it's `webVitals` or `web.vitals` or `webvitals`) so that users don't have to do that?

Reply:
`web.vitals` works 👍